docs/examples/settings_main.py

```diff
@@ -21,6 +22,7 @@ class Settings(BaseSettings):
 
     redis_dsn: RedisDsn = 'redis://user:pass@localhost:6379/1'
     pg_dsn: PostgresDsn = 'postgres://user:pass@localhost:5432/foobar'
+    rabbitmq_dsn: RabbitmqDsn = 'amqp://guest:guest@rabbitmq:5672//'
```

Comment:
Is the double-slash at the end intended?

Reply:
Yes - I'm not 100% sure why but from what I've read it allows for matching of any vhost - https://www.rabbitmq.com/uri-spec.html 

![image](https://user-images.githubusercontent.com/5226462/99201278-553a6980-27fe-11eb-8724-23fae3332249.png)
Where `%2F` = `/`

Honestly this was what was in my config of the web app that I'm working on - and I simply coppied and pasted.

Reply:
Also this is currently what I'm doing in my config - not sure it will help but I thought I'd share.
```
    CELERY_BROKER_SCHEME: str = "amqp"
    CELERY_BROKER_USERNAME: str = "guest"
    CELERY_BROKER_PASSWORD: str = "guest"
    CELERY_BROKER_HOST: str = "rabbitmq"
    CELERY_BROKER_PORT: int = 5672
    CELERY_BROKER_URI: Optional[AnyUrl] = None

    @validator("CELERY_BROKER_URI", pre=True)
    def assemble_rabbitmq_connection(cls, v: Optional[str], values: Dict[str, Any]) -> Any:
        if isinstance(v, str):
            return v
        uri = AnyUrl.build(
            scheme=values.get("CELERY_BROKER_SCHEME"),
            user=values.get("CELERY_BROKER_USERNAME"),
            password=values.get("CELERY_BROKER_PASSWORD"),
            host=values.get("CELERY_BROKER_HOST"),
            port=str(values.get("CELERY_BROKER_PORT")),
            path="//",
        )
        return uri
```

Reply:
Cool. I'm not familiar with rabbitmq, so this explains it. Thanks!

Reply:
I think we need a brief comment explaining that the double slash isn't a mistake, and what it means.